src/game/client/components/console.cpp

```diff
@@ -136,6 +136,8 @@ void CGameConsole::CInstance::PossibleArgumentsCompleteCallback(int Index, const
 	}
 }
 
+char pConPrompt[32];
```

Comment:
- you shouldn't use global variables for this
- this variable will be used uninitialized, so it will cause game crashes

Reply:
What would be better over global variables?